resources/views/project/ctest-configuration.blade.php

```diff
@@ -9,7 +9,13 @@
 set(CTEST_PROJECT_NAME {{ $project->Name }})
 set(CTEST_NIGHTLY_START_TIME {{ $project->NightlyTime }})
 
-set(CTEST_SUBMIT_URL {{ env('APP_URL') }}/submit.php?project={{ urlencode($project->Name) }})
+if(CMAKE_VERSION VERSION_GREATER 3.14)
+  set(CTEST_SUBMIT_URL {{ env('APP_URL') }}/submit.php?project={{ urlencode($project->Name) }})
```

Comment:
It's better to use `config('app.url')` here instead of directly querying an environment variable.  We want to limit our direct environment variable accesses to the configuration code if at all possible.

I think I was the original author of this code, but I didn't know the difference between `env()` and `config()` when I wrote this..  Oops!

Reply:
Done.  Thanks again, @williamjallen